packages/default/scss/scheduler/_layout.scss

```diff
@@ -17,6 +17,43 @@
         > * {
             border-color: inherit;
         }
+
+        // Current time
+
+        .k-current-time {
+            background: red;
+            position: absolute;
+        }
+
+        .k-current-time-arrow-down {
+            width: 0;
+            height: 0;
+            background: transparent;
+            border-bottom: 4px solid  transparent;
+            border-top: 4px solid #f00;
```

Comment:
Consider this color to be a variable or check if there is a existing variable for it